config/200-clusterrole-namespaced.yaml

```
@@ -0,0 +1,61 @@
+# Copyright 2022 TriggerMesh Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+kind: ClusterRole
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: triggermesh-namespaced-admin
+  labels:
+    rbac.authorization.k8s.io/aggregate-to-admin: "true"
+    app.kubernetes.io/part-of: triggermesh
+rules:
+- apiGroups:
+  - flow.triggermesh.io
+  - routing.triggermesh.io
+  - sources.triggermesh.io
+  - targets.triggermesh.io
+  resources: ["*"]
+  verbs: ["*"]
+---
+kind: ClusterRole
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: triggermesh-namespaced-edit
+  labels:
+    rbac.authorization.k8s.io/aggregate-to-edit: "true"
+    app.kubernetes.io/part-of: triggermesh
+rules:
+- apiGroups:
+  - flow.triggermesh.io
+  - routing.triggermesh.io
+  - sources.triggermesh.io
+  - targets.triggermesh.io
+  resources: ["*"]
+  verbs: ["create", "update", "patch", "delete"]
```

Same comment as before regarding
- mixing JSON and YAML syntaxes for lists containing several items
- superfluous quotes

```suggestion
  verbs:
  - create
  - update
  - patch
  - delete
```

---

Happy to adhere to this style. fwiw I took the original style from knative roles.